src/couch/entry.js

```diff
@@ -11,7 +11,7 @@ const validateMethods = require('./validate');
 
 const methods = {
   async getEntryWithRights(uuid, user, rights, options = {}) {
-    debug('getEntryWithRights (%s, %s, %o)', uuid, user, rights);
+    debug('getEntryWithRights (%s, %s, %o, %o)', uuid, user, rights, options);
```

Comment:
```suggestion
    debug('getEntryWithRights (%s, %s, %o)', uuid, user, rights);
```

Printing the entire token is too much for a regular debug